### DISCUSSION TITLE
Support for using Livechart2 in a project

### DISCUSSION THREAD

**Discussion by Chinmay062001** [unoplatform/uno#18137] [General]
Please provide a functional git repo using Livechart2 for charting and MapControl in UNO setup for cross-platform.
Specs - UNO5, WinUI3,.net8
@jeromelaban If possible could u assist me in this.

**Comment by jeromelaban** [comment]
Thanks for the question, you can view all our samples here: https://github.com/unoplatform/uno.samples
